tests/docker-images/statestore-image/image_builder.sh

```diff
@@ -18,22 +18,34 @@
 # * limitations under the License.
 # */
 
+IMAGE_NAME=apachebookkeeper/bookkeeper-current:latest
```

Comment:
One problem I see with this optimization is that, when debugging or locally running test, I would not know for sometime why changes I made is not taking affect, until I see that image_builder is not building new image anymore.